### ISSUE TITLE
Missing Icons

### ISSUE THREAD

**Issue by Omzig** [ironmansoftware/powershell-universal#1753]
### Steps to Reproduce

Just one example, is search for dice:

```powershell
New-UDTextbox -Id 'txtIconSearch' -Label 'Search' 
New-UDButton -Text 'Search' -OnClick {
    Sync-UDElement -Id 'icons'
}

New-UDElement -tag 'p' -Content {}

New-UDDynamic -Id 'icons' -Content {
    $IconSearch = (Get-UDElement -Id 'txtIconSearch').value
    if ($null -ne $IconSearch -and $IconSearch -ne '')
    {
        $Icons =$Icons = Find-UDIcon -Name $IconSearch
    }

    foreach($icon in $icons) {
        try{
            New-UDChip -Label $icon -Icon (New-UDIcon -Icon $icon)
        }
        catch{
            New-UDChip -Label "$icon Unknown" 
        }
    }
}
```

### Expected behavior

```console
0 missing icons
```

### Actual behavior

```console
randomly missing icons
```

### Environment data

PSU: 3.5.5

### Visuals

![image](https://user-images.githubusercontent.com/1987296/203457638-caaaa934-0793-4f5e-b755-e88069cafaba.png)

**Comment by Omzig** [comment]
@adamdriscoll, By any chance, will this fix my issue in 3.6.0?
- https://github.com/ironmansoftware/issues/issues/1715

**Comment by adamdriscoll** [comment]
@Omzig - It doesn't. What I'm realizing now is that the reason there are missing icons is because some of the icons are free in one format (solid) and pro for other formats (regular).

**Comment by adamdriscoll** [comment]
I think we should just get the pro icons...

**Comment by Omzig** [comment]
> I think we should just get the pro icons...

the licensing on that makes my head hurt.

**Comment by rwidmark** [comment]
> I think we should just get the pro icons...

Sounds great!

**Comment by rwidmark** [comment]
@adamdriscoll have you added all of the icons?

**Comment by adamdriscoll** [comment]
@rstolpe - Yep. All pro icons will be available in 3.8.
